packages/DataTable/src/components/Navigation/components/Filters/FilterItem.js

```diff
@@ -57,6 +57,14 @@ export default function FilterItem(prop) {
     });
   }
 
+  function handleChangeDatePicker(momentDate) {
+    if (momentDate === null) {
+      handleChangeValue("");
+    } else {
+      handleChangeValue(momentDate.format(columns[selectedColumnId].momentParsingFormat));
```

Comment:
🤔 what if they pass an incorrect momentParsingFormat? shouldn't we check if the date produced by moment its valid?

Reply:
I think date picker already handled the case when users type invalid date, I'll double-check